src/test/java/seedu/address/logic/parser/people/PeopleReturnedCommandParserTest.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.parser.people;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_TRANSACTION_INDEX;
+import static seedu.address.logic.commands.CommandTestUtil.RETURNED_DESC_AMY;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.people.PeopleReturnedCommand;
+
+public class PeopleReturnedCommandParserTest {
+
+    private static final String MESSAGE_INVALID_FORMAT =
+            String.format(MESSAGE_INVALID_COMMAND_FORMAT, PeopleReturnedCommand.MESSAGE_USAGE);
+
+    private PeopleReturnedCommandParser parser = new PeopleReturnedCommandParser();
+
+    @Test
+    public void parse_missingParts_failure() {
+        // nothing specified
+        assertParseFailure(parser, " ", MESSAGE_INVALID_FORMAT);
+
+        // no person's index specified
+        assertParseFailure(parser, "i/1", MESSAGE_INVALID_FORMAT);
+    }
+
+    @Test
+    public void parse_invalidPreamble_failure() {
+        // negative index
+        assertParseFailure(parser, "-5" + RETURNED_DESC_AMY, MESSAGE_INVALID_FORMAT);
+
+        // zero index
+        assertParseFailure(parser, "0" + RETURNED_DESC_AMY, MESSAGE_INVALID_FORMAT);
+
+        // invalid arguments being parsed as preamble
+        assertParseFailure(parser, "1 some random string" + RETURNED_DESC_AMY, MESSAGE_INVALID_FORMAT);
+
+        // invalid prefix being parsed as preamble
+        assertParseFailure(parser, "1 p/ string" + RETURNED_DESC_AMY , MESSAGE_INVALID_FORMAT);
+    }
+
+    @Test
+    public void parse_invalidValue_failure() {
+        // invalid person index
```

Comment:
can add invalid debt index here as well

Reply:
Sorry this test is invalid debt index. I put wrong comment haha. invalid person index is handled in invalidPreamble above